### PR TITLE
Clean up and comment a bunch of stuff

### PR DESCRIPTION
**Pull request by ArthurAllshire**

### CODE REVIEW

**Comment by ArthurAllshire** [comment]
Fixes #2 
Fixes #7

File: automations/range_filter.py [hunk 1]
```diff
@@ -34,126 +34,95 @@ class RangeFilter:
     vision_based_range_variance = 0.1
 
     def __init__(self):
-        pass
+        self.logger = logging.getLogger("Range Filter")
```
Comment (auscompgeek) [inline automations/range_filter.py]: This is unnecessary.

File: automations/vision_filter.py [hunk 2]
```diff
@@ -32,31 +32,44 @@ class VisionFilter:
     control_loop_average_delay = MagicRobot.control_loop_wait_time/2
 
     def __init__(self):
-        pass
+        # crete the state transition matrix
```
Comment (auscompgeek) [inline automations/vision_filter.py]: nit: typo

File: automations/vision_filter.py [hunk 3]
```diff
@@ -32,31 +32,44 @@ class VisionFilter:
...
+        self.F = np.identity(self.state_vector_size)
+        self.F[0][1] = self.loop_dt
```
Comment (auscompgeek) [inline automations/vision_filter.py]: Use numpy ndarray index syntax please. This also goes for everywhere else.

**Comment by ArthurAllshire** [comment]
Made changes requested in review.

File: automations/range_filter.py [hunk 4]
```diff
@@ -12,15 +12,15 @@
 
 from collections import deque
 
+import logging
```
Comment (auscompgeek) [inline automations/range_filter.py]: **F401** 'logging' imported but unused

File: automations/range_filter.py [hunk 5]
```diff
@@ -12,15 +12,15 @@
 
 from collections import deque
```
Comment (auscompgeek) [inline automations/range_filter.py]: **F401** 'collections.deque' imported but unused

**Pull request merged**